packages/aquedux-server/src/managers/configManager.js

```diff
@@ -1,25 +1,54 @@
 // @flow
 
-type InternalState = {
-  reduceStateToQueueSnapshot: ?(any) => any
-}
+import keygen from 'keygenerator'
 
-const internalState: InternalState = {
-  reduceStateToQueueSnapshot: null
-}
+import logger from '../utils/logger'
+
+let config = {
+  queueLimit: 0,
+  statefullTypes: [],
+  routePrefix: '/aquedux',
+  /**
+   * A new JWT secret is generated at each start.
+   * User should override it with a contant one
+   * if he needs to persist some JWT token uppon server restart
+   * or client reconnection on a different server
+   */
+  secret: keygen.password(),
+
+  redisHost: process.env.DB_PORT_6379_TCP_ADDR || '127.0.0.1', // Default redis env var
+  redisPort: process.env.DB_PORT_6379_TCP_PORT || '6379', // Default redis env var
 
-const setReduceStateToQueueSnapshot = (cb: any => any) => {
-  internalState.reduceStateToQueueSnapshot = cb
+  doFragmentSnapshot: (prevState: any): any => prevState,
+
+  onConnection: (_socket: any) => {},
+  onClose: (_socket: any) => {}
 }
 
-const reduceStateToQueueSnapshot = (prevState: any): any => {
-  if (!internalState.reduceStateToQueueSnapshot) {
-    return prevState
-  }
-  return internalState.reduceStateToQueueSnapshot(prevState)
+export type AqueduxConfig = typeof config
+
+const setConfig = (newConfig: any): AqueduxConfig => {
+  config = Object.keys(newConfig).reduce((result, key) => {
+    if (!config.hasOwnProperty(key)) {
+      return result
```

Comment:
It would be awesome for the aquedux user to have a console error here that tells him his error (misspelled configuration key) like `"The configuration key ${key} is not supported. Check createAqueduxServer arguments"`.